components/director/cmd/main.go

```diff
@@ -33,9 +37,12 @@ type config struct {
 		Name     string `envconfig:"default=postgres,APP_DB_NAME"`
 		SSLMode  string `envconfig:"default=disable,APP_DB_SSL"`
 	}
-	APIEndpoint           string `envconfig:"default=/graphql"`
-	TenantMappingEndpoint string `envconfig:"default=/tenant-mapping"`
-	PlaygroundAPIEndpoint string `envconfig:"default=/graphql"`
+	APIEndpoint                   string `envconfig:"default=/graphql"`
+	TenantMappingEndpoint         string `envconfig:"default=/tenant-mapping"`
+	PlaygroundAPIEndpoint         string `envconfig:"default=/graphql"`
+	ScopesConfigurationFile       string
```

Comment:
`EnableScopesChecking` is a short-lived flag, required only for a transition phase, then scopes will be always checked. Because of that, it is better to make this parameter required from the beginning.